translations/es/LC_MESSAGES/messages.po

```diff
@@ -223,7 +228,7 @@ msgstr ""
 
 #: /content/includes/home.yaml
 msgid "Building the future web, together."
-msgstr "Armando el futuro de la web, juntos."
+msgstr "Juntos, Armando el futuro de la web."
```

Comment:
Okay, I'll merge in and then you can 1) Fix the grammar and 2) Add translation for "Brand Guidelines" - which is all super helpful!